src/components/pages/Animations/FaceoffReveal.js

```diff
@@ -44,6 +45,7 @@ const FaceoffReveal = props => {
   let userName = dynamicInfo.Submission1.Name;
   let opponentName = dynamicInfo.Submission2.Name;
   console.log('userName', userName, opponentName);
```

Comment:
this variable was not being used - instead of removing it I asked the linter to ignore it as it may be needed in the future.

Reply:
If the var is not in used, why not comment it out instead of adding an eslint override?

Reply:
this is good advice - will update PR now